docs/using-bi-for-auth.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Using Beyond Identity for Authentication
+sidebar_position: 2
+---
+In this guide we will show you how to create a tenant, an application, configure the application, and how to integrate with the Beyond Identity Embedded SDK
+
+### Tenant Setup
+
+Create a tenant via the self signup form:
+
+https://console-us.beyondidentity.com/signup
+
+After creating a tenant, you will be redirected to an Admin Console for the newly created tenant.
+
+### Create a new Realm
+
+Click the drop down on the top left corner and add a new realm. [[Screenshot](./screenshots/NewRealm.jpg)] 
+
+### Create an Application
+
+- Click 'Applications' in the left menu
+- Click 'Create app'
+- Ensure to fill out values for display name, client type, and redirect uris
```

Comment:
I would expand this a bit more:
- Fill out the display name with whatever you want to name this app
- Select the client type:
  - (describe the client types down here)
- Set the redirect_uri to be a URL where you want to receive the authorization code and state. This can be either:
  - An app scheme or Universal URL / App Link if you're implementing this in a native application
  - A URL to any page in your web application